pkg/expapi/v1/types.go

```diff
@@ -79,7 +79,7 @@ type HorizontalPodAutoscalerSpec struct {
 	MaxCount int `json:"maxCount" description:"upper limit for the number of pods"`
 	// Target is the target average consumption of the given resource that the autoscaler will try to maintain by adjusting the desired number of pods.
 	// Currently two types of resources are supported: "cpu" and "memory".
-	Target TargetConsumption `json:"target"	description:"target average consumption of resource that the autoscaler will try to maintain by adjusting the desired number of pods"`
+	Target TargetConsumption `json:"target" description:"target average consumption of resource that the autoscaler will try to maintain by adjusting the desired number of pods"`
```

Comment:
Yes, just change a tab to a space. This error will generate when you use "go get golang.org/x/tools/cmd/vet". It will install the lastest go vet(version 1.5). It doesn't allow the tab in StrutctTag.  Go vet 1.4 will not generate this error.